src/composables/useSynthsSDK.ts

```diff
@@ -1,28 +1,47 @@
 import { ref, computed } from "vue"
+import { ethers } from "ethers"
 import Synths, { getRecentSynthData, getTotalMarketData } from "synths-sdk"
-import { defaultAssetsConfig } from "synths-sdk/dist/src/lib/config/index"
-import { Web3Provider } from "@ethersproject/providers"
+import { JsonRpcProvider } from "@ethersproject/providers"
+// TOOD Remove after testphase.
+import { defaultAssetsConfig, defaultTestAssetsConfig } from "synths-sdk/dist/src/lib/config/index"
 
 const loading = ref(true)
 const data = ref({})
-const synthsSDK = ref()
 const totalMarketData = ref()
 const recentSynthData = ref()
 const lspPortfolio = ref()
+const totalSynthsMinted = ref(0)
+const totalPortfolioValue = ref(0)
+let synthsSDK: Synths
 
 export function useSynthsSDK() {
     /**
      * @notice Initialize the synths-sdk and load market data.
      * @param provider The web3 provider instance.
      */
-    async function init(provider: Web3Provider) {
+    async function init(provider: JsonRpcProvider, networkId: number) {
         loading.value = true
 
-        const networkId: number = (await provider.getNetwork()).chainId
-        synthsSDK.value = await Synths.create({ ethersProvider: provider })
-        totalMarketData.value = await getTotalMarketData([networkId], defaultAssetsConfig)
-        recentSynthData.value = await getRecentSynthData(networkId, defaultAssetsConfig)
-        lspPortfolio.value = await synthsSDK.value.getLSPPortfolio(provider.getSigner())
+        const POLYSCAN_API_KEY = import.meta.env.VITE_POLYSCAN_API_KEY as string
+
+        synthsSDK = await Synths.create({ ethersProvider: provider, userAssetsConfig: defaultTestAssetsConfig })
+        totalMarketData.value = await getTotalMarketData([networkId], synthsSDK.config, POLYSCAN_API_KEY)
+        recentSynthData.value = await getRecentSynthData(networkId, synthsSDK.config, POLYSCAN_API_KEY)
```

Comment:
should be separate and selectable depending on the network we are on

Reply:
Not sure what you mean by that. It uses passes the network that is connected.

Reply:
`POLYSCAN_API_KEY`

Reply:
We only need the key for polyscan and as this is an optional param on the helpers and it will just be ignored if it is on mainnet i think we can keep it as is instead of adding a switch or if / else statement.

Reply:
for now thats ok since we will have everything on polygon, but for a later update just pass it with the `synthsSDK.config` values